examples/node/collections.js

```diff
@@ -37,13 +35,13 @@ async function publish() {
         sri: "sha256-zjQXDuk++5sICrObmfWqAM5EibidXd2emZoUcU2l5Pg="
       }
     ],
-    url: "https://google.com",
+    url: "https://example.com",
     publicKey: wallet.publicKey,
     destination: wallet.selectedAddress,
     changeAddress: wallet.selectedAddress,
     postage: 1000,
-    mediaContent: IMG_BASE64,
-    mediaType: "image/png"
+    mediaContent: 'Collection Name', // this will be inscribed on-chain as primary content
```

Comment:
Is there a reason this string is surrounded by single quotes?

Reply:
Looks like this inconsistency is present through the repo.

Reply:
Linter doesn't work in the `/examples` dir. Something to work on later.